### PR TITLE
Basic Main Menu

### PR DESCRIPTION
**Pull request by DuaneShaffer**
Merge in partial main menu so that others can use it. More development is to be done to create the rest of the buttons and make the look right

### CODE REVIEW

File: website/src/components/MainMenu.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                          block
+                          large
+                          color=rgba(21,38,112)
+                          dark
+                          v-on:click="$emit('createOrder')">
+                          Create Order
+                      </v-btn>
+              </v-flex>
+              <v-flex>
+                      <v-btn
```
Comment (williamh890) [inline website/src/components/MainMenu.vue]: Probably can condense these btns to 2 or 3 lines.

File: website/src/store/orders/mutation.js [hunk 2]
```diff
@@ -47,6 +58,7 @@ export const mutations = {
   },
 
   [SET_ORDER_DATE]: function(state, date) {
+    console.log(state.order.date.toString());
```
Comment (williamh890) [inline website/src/store/orders/mutation.js]: No Logs!!

File: website/src/components/MainMenu.vue (hunk 1 above)
Comment (DuaneShaffer) [reply website/src/components/MainMenu.vue]: Just because I can doesn't mean I should.

Comment (DuaneShaffer) [reply website/src/components/MainMenu.vue]: Also, remind me that we need to discuss changing the "primary" color today.

File: website/src/store/orders/mutation.js (hunk 2 above)
Comment (DuaneShaffer) [reply website/src/store/orders/mutation.js]: I get rid of that in another PR... the beauty of having all of them stack up

**Pull request merged**